Traceback (most recent call last):
  File "/lfs/l1/legend/users/aalexander/HADES_detchar/Ba133_analysis/simulations/IC-legend/macro/ba_top/PostProc/postprochdf5.py", line 56, in <module>
    main()
  File "/lfs/l1/legend/users/aalexander/HADES_detchar/Ba133_analysis/simulations/IC-legend/macro/ba_top/PostProc/postprochdf5.py", line 30, in main
    g4sfile = h5py.File(hdf5_path+'detector_'+MC_file_id+'.hdf5', 'r')
  File "/home/extern/aalexander/miniconda3/lib/python3.7/site-packages/h5py/_hl/files.py", line 408, in __init__
    swmr=swmr)
  File "/home/extern/aalexander/miniconda3/lib/python3.7/site-packages/h5py/_hl/files.py", line 173, in make_fid
    fid = h5f.open(name, flags, fapl=fapl)
  File "h5py/_objects.pyx", line 54, in h5py._objects.with_phil.wrapper
  File "h5py/_objects.pyx", line 55, in h5py._objects.with_phil.wrapper
  File "h5py/h5f.pyx", line 88, in h5py.h5f.open
OSError: Unable to open file (unable to open file: name = '/lfs/l1/legend/users/aalexander/hdf5_output/detector_IC160A_ba_top_81mmNEW2.hdf5', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0)
Traceback (most recent call last):
  File "/lfs/l1/legend/users/aalexander/HADES_detchar/Ba133_analysis/simulations/IC-legend/macro/ba_top/PostProc/postprochdf5.py", line 55, in <module>
    main()
  File "/lfs/l1/legend/users/aalexander/HADES_detchar/Ba133_analysis/simulations/IC-legend/macro/ba_top/PostProc/postprochdf5.py", line 29, in main
    g4sfile = h5py.File(hdf5_path+'detector_'+MC_file_id+'.hdf5', 'r')
  File "/home/extern/aalexander/miniconda3/lib/python3.7/site-packages/h5py/_hl/files.py", line 408, in __init__
    swmr=swmr)
  File "/home/extern/aalexander/miniconda3/lib/python3.7/site-packages/h5py/_hl/files.py", line 173, in make_fid
    fid = h5f.open(name, flags, fapl=fapl)
  File "h5py/_objects.pyx", line 54, in h5py._objects.with_phil.wrapper
  File "h5py/_objects.pyx", line 55, in h5py._objects.with_phil.wrapper
  File "h5py/h5f.pyx", line 88, in h5py.h5f.open
OSError: Unable to open file (unable to open file: name = '/lfs/l1/legend/users/aalexander/hdf5_output/detector_detector_IC160A_ba_top_81mmNEW4_01.hdf5', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0)
/var/spool/SGE/lf/lfc103/job_scripts/8576464: line 5: 15621 Killed                  python /lfs/l1/legend/users/aalexander/HADES_detchar/Ba133_analysis/simulations/IC-legend/macro/ba_top/PostProc/postproc_FCCD.py IC160A_ba_top_81mmNEW3_01
/var/spool/SGE/lf/lfc309/job_scripts/8603133: line 5: 25758 Killed                  python /lfs/l1/legend/users/aalexander/HADES_detchar/Ba133_analysis/simulations/IC-legend/macro/ba_top/PostProc/postproc_FCCD.py IC160A_ba_top_81mmNEW4_01
